lib/SILOptimizer/Utils/InstOptUtils.cpp

```diff
@@ -701,41 +701,58 @@ SILValue swift::
 getConcreteValueOfExistentialBox(AllocExistentialBoxInst *existentialBox,
                                   SILInstruction *ignoreUser) {
   StoreInst *singleStore = nullptr;
-  for (Operand *use : getNonDebugUses(existentialBox)) {
+  SmallVector<Operand *, 32> worklist;
+  SmallPtrSet<Operand *, 8> addedToWorklistPreviously;
```

Comment:
We use `SmallSetVector` everywhere else for this pattern. If it's worth it to do this then we should add a thin `SmallPtrSetVector` wrapper and use that everywhere instead

Reply:
I think SmallSetVector already uses a SmallPtrSet when it has a pointer operand. So it should just work. I think I just wrote this quickly.